Course/Course/Presentation/Video/YoutubePlayerViewControllerHolder.swift

```diff
@@ -49,23 +49,23 @@ public class YoutubePlayerViewControllerHolder: PlayerViewControllerHolderProtoc
         blockID: String,
         courseID: String,
         selectedCourseTab: Int,
-        videoResolution: CGSize,
         pipManager: PipManagerProtocol,
         playerTracker: any PlayerTrackerProtocol,
         playerDelegate: PlayerDelegateProtocol?,
-        playerService: PlayerServiceProtocol
+        playerService: PlayerServiceProtocol,
+        appStorage: CoreStorage?
     ) {
         self.url = url
         self.blockID = blockID
         self.courseID = courseID
         self.selectedCourseTab = selectedCourseTab
-        self.videoResolution = videoResolution
         self.pipManager = pipManager
         self.playerTracker = playerTracker
         self.playerService = playerService
         let youtubePlayer = playerTracker.player as? YouTubePlayer
         var configuration = youtubePlayer?.configuration
         configuration?.autoPlay = !pipManager.isPipActive
+        configuration?.fullscreenMode = .web
```

Comment:
@rnr Is this change relevant?

Reply:
@saeedbashir it's not relevant to playback speed. It adds fullscreen for iPad. Update PR description.
Thank you